vendor/font-awesome/css/font-awesome.css

```diff
@@ -1278,6 +1278,9 @@
 .fa-male:before {
   content: "\f183";
 }
+.fa-gitter:before {
+content:"\eaa4";
+}
```

Comment:
I don't really know what's happening here, but we probably shouldn't modify the original font-awesome css file (the changes will get lost when upgrading the package). You could add custom styles in freelancer.less instead.